docs/expansionPort.md

```diff
@@ -0,0 +1,115 @@
+
+
+| Parameter                             |    Conditions     |   MIN   |  MAX  |
+|---------------------------------------|-------------------|---------|-------|
+| Supply voltage VDD (Differential side)|                   | 2.7 V   | 5.5 V |
+| IO supply voltage VDDS (SPI side)     |                   | 1.7 V   | 5.5 V |
+| High-level input voltage              |VDDS = 2.7V to 5V  | 0.7 VDDS| VDDS  |
+| High-level input voltage              |VDDS = 1.7V to 2.7V| 0.8 VDDS| VDDS  |
+| Low-level input voltage               |VDDS = 2.7V to 5V  | 0       | 0.3 VDDS  |
+| Low-level input voltage               |VDDS = 1.7V to 2.7V| 0       | 0.2 VDDS  |
+
+This IC can operate at a maximum SPI communication speed of 1 Mbps. Bias resistors (RB1 and RB2) are used to adjust the drive currents to the differential lines, in this design they set the drive currents to 10 mA. These resistors are on the schematic, see the following figure.
+
+<img src="images/Schematic_RB.PNG" width="200"/>
+
+The maximum supply current consumed by the IC including to drive currents for differential lines is 15.8 mA, which corresponds to 79 mW for a 5 V supply. More information regarding the operating conditions, bias resistors, maximum throughput rate, can be found in the [datasheet](https://www.analog.com/media/en/technical-documentation/data-sheets/LTC6820.pdf). 
+
+
+### 3. IsoSPI isolation transformer
+
+To isolate the differential isoSPI signals, the HX1188NLT pulse transformer is used. More information on the pulse transformer is found in the [datasheet](https://media.digikey.com/pdf/Data%20Sheets/Pulse%20PDFs/10_100BASE-T%20Single%20Port%20SMD%20Magnetics_Rev2008.pdf).
+
+
+### 4. Single-ended to differential converter
+
+The I/O interface is connected to the FPGA I/O pins of the PicoZed, which can be configured as SPI, UART, or any other communication interface. This signal is converted into differential I/O using the AM26C31 differential line driver. The operating conditions for the IC are provided in the following table.
+
+| Parameter                             | MIN   | NOM | MAX   |
+|---------------------------------------|-------|-----|-------|
+| Supply voltage Vcc                    | 4.5 V | 5 V | 5.5 V |
+| Input voltage range                   | -0.5 V|     | Vcc + 0.5 V  |
+| High-level input voltage              | 2 V   |     | Vcc   |
+| Low-level input voltage               | 0 V   |     | 0.8 V |
+
+The maximum supply current consumed by the IC including the drive currents for the differential lines is around 20 mA, which corresponds to 100 mW for a 5 V supply. For more information refer to the [AM26C31](http://www.ti.com/lit/ds/symlink/am26c31.pdf?HQS=TI-null-null-digikeymode-df-pf-null-wwe&ts=1590045318995) datasheet.
+
+
+### 5. Differential to single-ended converter

```

Comment:
What data throughput can we expect to achieve with differential I/O? Any guidance on cable length? (Long cables? short cables?)

Reply:
I have added information on data throughput. But I couldn't find information on cable length from the differential line converter IC.